drupal8-apache/7.0/Dockerfile

```diff
@@ -8,7 +8,6 @@ RUN curl -q https://dx6pc3giz7k1r.cloudfront.net/GPG-KEY-inviqa-tools | apt-key
  && bash /tmp/install-node.sh \
  && apt-get update -qq -y \
  && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install \
-    hem \
```

Comment:
No drupal projects currently use hem, so no concerns from me. There's more that can be removed though